src/app/app.component.ts

```diff
@@ -20,6 +21,10 @@ export class AppComponent implements OnInit {
 
   places: Place[] = [];
   currentPaginationPage = DEFAULT_PAGINATION_PAGE;
+  lowValue = 0;
```

Comment:
`lowValue` and `highValue` can be private because they are used only in this class?